test/embedding/README.md

```diff
@@ -0,0 +1,124 @@
+# C embedding API
+
+This file is an overview for C embedding API.
```

Comment:
Does this comment stil make sense?

Reply:
or maybe it more that the who README.md needs an update, given that there is documentation elsewhere?

Reply:
Yeah, I must update this file. I used it as a scratch pad for some thoughts and TODOs.